network/router.go

```diff
@@ -22,7 +22,7 @@ import (
 //   router.Stop() // will stop the listening and the managing of all Conn
 type Router struct {
 	// id is our own ServerIdentity
-	id *ServerIdentity
+	ServerIdentity *ServerIdentity
```

Comment:
IMO it is related to this issue: `Send` and `Connect` only took `Address` before, but that is not enough with TLS, we need the `Cert` which is in `ServerIdentity`.